force-app/main/default/objects/npe01__OppPayment__c/fields/DebitType__c.field-meta.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
-    <fullName>RefundType__c</fullName>
-    <description>Determine if the record is fully or partially refunded</description>
+    <fullName>DebitType__c</fullName>
+    <description>Indicates the type of debit.</description>
     <externalId>false</externalId>
-    <inlineHelpText>Determine if the record is fully or partially refunded</inlineHelpText>
-    <label>Refund Type</label>
+    <inlineHelpText></inlineHelpText>
```

Comment:
No inline help text for the new field?

Reply:
Nope, the field name and values seemed clear enough that we don't need help text.